src/utils/ethereumUtils.ts

```diff
@@ -524,19 +557,36 @@ async function parseEthereumUrl(data) {
   });
 }
 
-const calculateL1FeeOptimism = async (tx, provider) => {
+const calculateL1FeeOptimism = async (
+  tx: RainbowTransaction,
+  provider: Provider
+) => {
```

Comment:
Based on `parseLegacyGasFeesBySpeed` where the return value ends up being used, I think this returns `BigNumber | undefined`, but could be wrong.